From 2d4e75711d2212d808bb167b0b3c8d9eecb7d7f1 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Tue, 5 Jun 2018 13:29:12 -0400
Subject: [PATCH] TRITON-461 triton services still on nodejs 0.10.x should at
 least use the latest x TRITON-464 sdc-napi-ufds-watcher NODE_PREBUILT_IMAGE
 does not match runtime environment Reviewed by: Trent Mick <trentm@gmail.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 Makefile | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 4389e08..7ebe6d2 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -37,11 +37,10 @@ JSSTYLE_FLAGS    = -o indent=4,doxygen,unparenthesized-return=0
 JSON_FILES  := package.json
 SMF_MANIFESTS_IN = smf/manifests/napi-ufds-watcher.xml.in
 
-NODE_PREBUILT_VERSION=v0.10.29
+NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
-	# Allow building on a SmartOS image other than sdc-smartos@1.6.3.
-	NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
 
 include ./tools/mk/Makefile.defs
-- 
2.21.0

